aws/dms/tasks.yml

```diff
@@ -52,13 +52,25 @@ cron-pii:
 - dashboard.learning_goal_teacher_evaluations
 - dashboard.learning_goal_ai_evaluation_feedbacks
 cron:
+- dashboard.ai_tutor_interactions:
+    remove_column:
+      - prompt
+      - ai_response
```

Comment:
These are the 2 fields we most need 🙃. We want share this data with Product to be able to analyze the accuracy/utility/pedagogical alignment of the AI responses to know if we're iterating in the right direction. How risky is it to include these columns in what gets sent to Redshift?

Reply:
Risky in what sense?

Reply:
Looks like this PR is removing those columns from the transfer because they could potentially contain PII. I'd like to to better understand why that's problematic if the people who are accessing the data via Redshift are internal to Code.org. Like how is Redshift worse from a privacy/security perspective than me manually pulling the data from prod and putting it in a csv to share with the AI Tutor team?

Reply:
We should have as few copies of PII as possible, and we should ensure that PII is always categorized as such. If it's not needed, then lets not make a copy of it, if it is needed, then lets make sure it stays labeled as PII.

If we need the `prompt` and `ai_response` fields in Redshift, then we just need to move this to the cron_pii schema.

Reply:
I've updated this configuration file to also export this table with all columns to the `dashboard_production_pii` schema. Let us know if this meets your needs. We seem to have a pattern of exporting a table to both schema, with the sensitive columns excluded when exporting to the non-PII schema.